lib/season_statistics.rb

```diff
@@ -69,9 +103,23 @@ def fewest_tackles
         end
            
             team_fewest_tackles = tackles_teams.min_by { |team, tackles| tackles }[0]
-    end
-
+    
+            find_team_name(team_fewest_tackles)
+        end
+        def find_team_name(team_id)
+            #get the corresponding team name to the team id number in teams...
+        end
     def most_tackles
+        tackles_teams = Hash.new(0)
+        @game_data.each do |game|
```

Comment:
your game_data will probably have to be changed to @game_teams or @games with the new code